### PR TITLE
Feature/input-component size and accessibility stories

### PR DESCRIPTION
**Pull request by Naseer-007**
- Styled input component for large and small sizes matching designs
- Added text-area for normal input and its missing styles
- Created 2 different mixin's, one for small size and one for large size
- Added config size to ngClass
- Added simple logic for which specific mixin should be used in which scenario
- Added missing Aria-labels to match accessibility requirements

### CODE REVIEW

**Review by MikeJalow — APPROVED** [review]

**Review by kristopher-houston — APPROVED** [review]

File: component-library/component-lib/src/lib/input/input.component.html [hunk 1]
```diff
@@ -20,14 +20,21 @@
         <div class="input-content-area" [ngClass]="focusState === true ? 'focus' : ''">
           <!-- ngModel doesn't mind undefined values, apparently
           TODO: Should probably change the (keyup) to (onblur) -->
-          <input class="input-field" placeholder="{{ (config.placeholder || '') | translate }}"
+          <textarea *ngIf="(config.type === 'text' || !config.type) && !showPassword" class="input-text-field" [id]="config.id"
```
Comment (kristopher-houston) [inline component-library/component-lib/src/lib/input/input.component.html:23]: Why was this changed to textarea? This has other implications....

Comment (Naseer-007) [reply component-library/component-lib/src/lib/input/input.component.html:23]: So according to designs the text input field has multiple lines/rows so for that to work text-area seemed to be a good solution

Comment (Naseer-007) [reply component-library/component-lib/src/lib/input/input.component.html]: After checking with Tyh, the text-area field will be a separate story and this story only covers single line input, so removed the text-area and added back input for single-line non password inputs
